openshift/imagestreams/jenkins-rhel.yaml

```diff
@@ -47,7 +47,7 @@ spec:
       version: 2.x
     from:
       kind: DockerImage
-      name: registry.redhat.io/ocp-tools-4/jenkins-rhel8:v4.11.0-1663086392.1666642635
+      name: registry.redhat.io/ocp-tools-4/jenkins-rhel8:v4.12.0-1663086392.1666642635
```

Comment:
@coreydaley  where does this come from ?
I cant see the 4.12 image in the redhat catalog https://catalog.redhat.com/software/containers/ocp-tools-4/jenkins-rhel8/5fe1f38288e9c2f788526306

Reply:
Search & Replace error, will fix in next push

Reply:
@coreydaley we dont have any 4.12 image, so i tink we may have to do the bump of the image stream refrences  in a separate PR, because this will be used by sample/operator
I suggest we do the sync with master here without bumping any of the image streams

Reply:
updated in current push